Examples/CommonSwiftCode/PageViewController.swift

```diff
@@ -48,9 +48,21 @@ class PageViewController:  UIPageViewController, UIPageViewControllerDelegate, U
             builder!.setApplicationContext(true)
             builder!.setExceptionEvents(true)
             builder!.setInstallEvent(true)
+            builder!.setGlobalContextGenerators(["globalContextTag1": self.getGlobalContext()])
         })
         return newTracker!
     }
+    
+    func getGlobalContext() -> SPGlobalContext {
```

Comment:
This is a good complete example (containing a Ruleset) but it is quite a complex example. Might it be worth, in addition to this example, adding a static Global Context - one that is on every event?